frontend/amundsen_application/static/js/components/ToggleSwitch/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+export * from './ToggleSwitch';
```

Comment:
Is there a reason this is in a separate file? Seems like the pattern for others would be to just to have the component defined in ToggleSwitch/index.tsx directly

Reply:
This pattern is preferable going forward. It allows you to have the same import for simple and complex (multi-component) components.